### PR TITLE
feat: 🚀 Implemented base folder-view design

### PR DESCRIPTION
**Pull request by gpuente**
## Description:

- Implemented base folder-view design

Note: This PR depends on https://github.com/powerhouse-inc/design-system/pull/128

### CODE REVIEW

File: src/components/folder-view.tsx [hunk 1]
```diff
@@ -33,54 +40,52 @@ export const FolderView: React.FC<IProps> = ({
...
+                {files.length > 0 ? (
+                    files.map(file => (
+                        <FileItem
                             key={file.id}
-                            className="cursor-pointer rounded-md hover:bg-black/5"
+                            title={file.name}
+                            subTitle={`MakerDAO/Ecosystem Actors/Powerhouse/${file.name}`}
+                            className="w-64"
+                            onOptionsClick={() =>
+                                onFileDeleted(decodedDriveID, file.id)
```
Comment (gpuente) [inline src/components/folder-view.tsx:76]: I'll keep the delete action attached to `onOptionsClick` for now, while we wait for the menu designs

File: src/components/content/section.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { DivProps } from '@powerhousedao/design-system';
+import React from 'react';
+
+export interface ContentSection extends DivProps {
+    title?: string;
+    children?: React.ReactNode;
```
Comment (ryanwolhuter) [inline src/components/content/section.tsx:6]: `children` is already part of DivProps, not important though

**Review by ryanwolhuter — APPROVED** [review]
nice work 👍

File: src/components/folder-view.tsx [hunk 3]
```diff
@@ -33,54 +40,52 @@ export const FolderView: React.FC<IProps> = ({
...
+                )}
+            </ContentSection>
+            <ContentSection title={t('folderView.sections.documents.title')}>
+                {files.length > 0 ? (
+                    files.map(file => (
+                        <FileItem
                             key={file.id}
-                            className="cursor-pointer rounded-md hover:bg-black/5"
+                            title={file.name}
+                            subTitle={`MakerDAO/Ecosystem Actors/Powerhouse/${file.name}`}
```
Comment (acaldas) [inline src/components/folder-view.tsx:73]: This should be the file path no?

Comment (gpuente) [reply src/components/folder-view.tsx]: you're right, here is the fix: [9f6a4ac](https://github.com/powerhouse-inc/document-model-electron/pull/46/commits/9f6a4ac45318bb757e7c7c60df463e67b066771b)

File: src/components/content/section.tsx (hunk 2 above)
Comment (gpuente) [reply src/components/content/section.tsx]: right! fix here: [9f6a4ac](https://github.com/powerhouse-inc/document-model-electron/pull/46/commits/9f6a4ac45318bb757e7c7c60df463e67b066771b)

**Pull request merged**